civihr_employee_portal/views/templates/views-view--default.tpl.php

```diff
@@ -27,6 +27,24 @@
  * @ingroup views_templates
  */
 ?>
+
+<?php
+  // Logic to change template for specific views
```

Comment:
it's not clear at all that this is temporary. Please provide more context in this comment to explain what is going on and especially *why*

Reply:
Done.